S21/Rongxiang-Zhang/lower.py

```diff
@@ -0,0 +1,60 @@
+import csv
+
+
+def wordlower(word,lan):
+
+    word=word
+
+    lan=lan
```

Comment:
We don't need to reassign the variables to the same name, we can just use the names we passed into the function. For example, 
def NamePrinter (name):
     print(f'Hello! {name}')

Reply:
Yes, you're right.